src/PrestaShopBundle/Utils/Migrate.php

```diff
@@ -39,10 +46,15 @@ public static function migrateSettingsFile(Event $event = null)
 
                 $default_parameters = Yaml::parse($root_dir.'/app/config/parameters.yml.dist');
 
+                if (defined('_LEGACY_RIJNDAEL_KEY_')) {
+                    $default_parameters['parameters']['_rijndael_key'] = _LEGACY_RIJNDAEL_KEY_;
+                    $default_parameters['parameters']['_rijndael_iv'] = _LEGACY_RIJNDAEL_IV_;
+                }
+
                 $parameters = array(
                     'parameters' => array(
                         'database_host'     => _LEGACY_DB_SERVER_,
-                        'database_port'     => '~',
+                        'database_port'     => '3306',
```

Comment:
ping @mickaelandrieu for confirmation.

If I remember well, this information is stored in the _LEGACY_DB_SERVER_, isn't it ?